Dev/subjects bug fix

notsecret32 opened the pull request.

notsecret32 committed:
fix: fixed typos on the student confirmation page

notsecret32 committed:
fix: the checkbox is now displayed

notsecret32 committed:
fix: reduced the width of the 'visiting-popover` window for lectures, practical and labs

notsecret32 committed:
feat: added subject deletion check

notsecret32 committed:
Merge branch 'develop' into dev/subjects-bug-fix

hzhurauski commented on modules/confirmation/src/app/confirmation/confirmation.component.html:
```diff
@@ -41,8 +41,8 @@
-        <th mat-header-cell *matHeaderCellDef>No.</th>
-        <td mat-cell *matCellDef="let index = index">{{ index + 1 }}</td>
+        <th mat-header-cell *matHeaderCellDef>№</th>
+        <td mat-cell *matCellDef="let index = index"> {{index+1}}</td>
```
Неверное форматирование

hzhurauski commented on modules/confirmation/src/app/core/translate/translate_ru.json:
```diff
@@ -1,11 +1,11 @@
-  "confirm.at": "Дата, время подтверждение",
-  "confirm.by": "Преподаватель"
-}
+    "students.search": "Поиск студентов",
```
Лишние пробелы

hzhurauski commented on modules/subjects/src/app/modules/subject/subject.component.ts:
```diff
@@ -129,28 +129,31 @@ export class SubjectComponent implements OnInit, OnDestroy {
       })
   }
 
-  deleteSubject(subject: Subject) {
```
Тут тоже проверить форматирование

notsecret32 committed:
fix: fixed formatting

notsecret32 committed:
fix: typo fixed

notsecret32 committed:
feat: added a new popover component

Warning Popover Component is a dialog component designed to display an OK dialog box

notsecret32 committed:
feat: added subject removal processing

Now you can't delete an subject that has a group

hzhurauski commented:
У тебя либо в корне проекта не все зависимости установлены, либо конфликт с локальным конфигом
